Cargo.toml

```diff
@@ -42,7 +42,7 @@ anyhow = "1.0.72"
 apache-avro = "0.17"
 array-init = "2"
 arrow-arith = { version = "53.3.0" }
-arrow-array = { version = "53.3.0" }
+arrow-array = { version = "53.4.0" }
```

Comment:
Seems like some of the versions are bumped to `53.4.0`, while others are still at `53.3.0`